packages/time-picker/test/value-commit.test.js

```diff
@@ -0,0 +1,391 @@
+      await sendKeys({ press: 'Enter' });
+      expectValidationOnly();
+      expect(timePicker.inputElement.value).to.equal('INVALID');
+    });
+
+    it('should not commit but validate on close with outside click', () => {
+      outsideClick();
+      expectValidationOnly();
+      expect(timePicker.inputElement.value).to.equal('INVALID');
+    });
+
+    it('should revert and validate on close with Escape', async () => {
+      await sendKeys({ press: 'Escape' });
+      await sendKeys({ press: 'Escape' });
+      expectValidationOnly();
+      expect(timePicker.inputElement.value).to.equal('');
+    });
+
+    describe('clearing input with Backspace', () => {
+      beforeEach(async () => {
+        await sendKeys({ press: 'Enter' });
+        valueChangedSpy.resetHistory();
+        validateSpy.resetHistory();
+
+        timePicker.inputElement.select();
+        await sendKeys({ press: 'Backspace' });
+      });
+
+      it('should not commit but validate on blur after clearing', () => {
+        timePicker.blur();
+        expectValidationOnly();
+      });
+
+      it('should not commit but validate on Enter after clearing', async () => {
+        await sendKeys({ press: 'Enter' });
+        expectValidationOnly();
+      });
+
+      it('should not commit but validate on close with outside click after clearing', () => {
+        outsideClick();
+        expectValidationOnly();
+      });
+
+      it('should revert and validate on close with Escape after clearing', async () => {
+        await sendKeys({ press: 'Escape' });
+        expectValidationOnly();
+        expect(timePicker.inputElement.value).to.equal('INVALID');
+      });
+    });
+  });
+
+  describe('overlay', () => {
+    beforeEach(async () => {
+      timePicker.click();
+      await sendKeys({ press: 'ArrowDown' });
+    });
+
+    it('should commit on selection with click', () => {
+      getAllItems(timePicker)[0].click();
+      expectValueCommit('00:00');
+    });
+
+    it('should commit on selection with Enter', async () => {
+      await sendKeys({ press: 'Enter' });
+      expectValueCommit('00:00');
+    });
+  });
+
+  describe('with value', () => {
+    let initialInputElementValue;
+
+    beforeEach(() => {
+      timePicker.value = '00:00';
+      initialInputElementValue = timePicker.inputElement.value;
+      valueChangedSpy.resetHistory();
+      validateSpy.resetHistory();
+    });
+
+    describe('default', () => {
+      it('should not commit but validate on blur', () => {
+        timePicker.blur();
+        expectValidationOnly();
+      });
+
+      it('should not commit but validate on Enter', async () => {
+        await sendKeys({ press: 'Enter' });
+        expectValidationOnly();
+      });
+
+      it('should not commit on Escape', async () => {
+        await sendKeys({ press: 'Escape' });
+        expectNoValueCommit();
+      });
+
+      it('should not commit but validate on close with outside click', async () => {
```

Comment:
nit: `async` is not needed here.

```suggestion
      it('should not commit but validate on close with outside click', () => {
```

Reply:
Fixed.